src/Python/Meshes/Decimation.py

```diff
@@ -0,0 +1,180 @@
+    parser.add_argument('filename', nargs='?', default=None, help='Optional input filename e.g cow.g.')
+    parser.add_argument('reduction', nargs='?', type = float, default=None, help='Optional reduction percentage e.g 0.9')
+    args = parser.parse_args()
+    return args.filename, args.reduction
+
+
+def main():
+    filePath, reduction = get_program_parameters()
+    
+    # Define colors
+    colors = vtk.vtkNamedColors()
+    backFaceColor = colors.GetColor3d("gold")
+    inputActorColor = colors.GetColor3d("flesh")
+    decimatedActorColor = colors.GetColor3d("flesh")
+
+    if filePath and os.path.isfile(filePath):
+        inputPolyData = ReadPolyData(filePath)
+        if not inputPolyData:
+            inputPolyData = GetSpherePD()
+    else:
+        inputPolyData = GetSpherePD()
+
+    if not reduction:
+        reduction = .9 # 90% reduction
+
+    print("Before decimation")
+    print(f"There are {inputPolyData.GetNumberOfPoints()} points.")
+    print(f"There are {inputPolyData.GetNumberOfPolys()} polygons.")
+
+    decimate = vtk.vtkDecimatePro()
+    decimate.SetInputData(inputPolyData)
+    decimate.SetTargetReduction(reduction)
+    decimate.PreserveTopologyOn()
+    decimate.Update()
+
+    decimated = vtk.vtkPolyData()
+    decimated.ShallowCopy(decimate.GetOutput())
+
+    print("After decimation")
+    print(f"There are {decimated.GetNumberOfPoints()} points.")
+    print(f"There are {decimated.GetNumberOfPolys()} polygons.")
+    print(f"Reduction: {(inputPolyData.GetNumberOfPolys() - decimated.GetNumberOfPolys()) / inputPolyData.GetNumberOfPolys()}")
+
+    inputMapper = vtk.vtkPolyDataMapper()
+    inputMapper.SetInputData(inputPolyData)
+
+    backFace = vtk.vtkProperty()
+    backFace.SetColor(backFaceColor)
+
+    inputActor = vtk.vtkActor()
+    inputActor.SetMapper(inputMapper)
+    inputActor.GetProperty().SetInterpolationToFlat()
+    inputActor.GetProperty().SetColor(inputActorColor)
+    inputActor.SetBackfaceProperty(backFace)
+
+    decimatedMapper = vtk.vtkPolyDataMapper()
+    decimatedMapper.SetInputData(decimated)
+
+    decimatedActor = vtk.vtkActor()
+    decimatedActor.SetMapper(decimatedMapper)
+    decimatedActor.GetProperty().SetColor(decimatedActorColor)
+    decimatedActor.GetProperty().SetInterpolationToFlat()
+    decimatedActor.SetBackfaceProperty(backFace)
+
+    # There will be one render window
+    renderWindow = vtk.vtkRenderWindow()
+    renderWindow.SetSize(600, 300)
+
+    # And one interactor
+    interactor = vtk.vtkRenderWindowInteractor()
+    interactor.SetRenderWindow(renderWindow)
+
+    # Define viewport ranges
+    # (xmin, ymin, xmax, ymax)
+    leftViewport = [0.0, 0.0, 0.5, 1.0]
+    rightViewport = [0.5, 0.0, 1.0, 1.0]
+
+    # Setup both renderers
+    leftRenderer = vtk.vtkRenderer()
+    renderWindow.AddRenderer(leftRenderer)
+    leftRenderer.SetViewport(leftViewport)
+    leftRenderer.SetBackground(.6, .5, .4)
+
```

Comment:
Use `vtkNamedColors` and you can do something like this:
``` Python
    colors.SetColor('leftBkg', [0.6, 0.5, 0.4, 1.0])
    colors.SetColor('rightBkg', [0.4, 0.5, 0.6, 1.0])
```
Here you have added some new named colors the colors that you can subsequently use.
Put these lines up where you define the colors, after line 28.
Then you can use `leftRenderer.SetBackground(colors.GetColor3d('leftBkg'))`